### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by megagnarxd** [Floris0106/Rereskillable-Rereforked#1]
1. Use EXP instead of levels like how it was implemented in reskillable instead of rereskillable
2. Please allow compatibility between this mod and epic fight mod.

Thank you very much :D

**Comment by Floris0106** [comment]
Dear god what are the github servers doing...

**Comment by Floris0106** [comment]
Anyway, can you elaborate on what you mean by "compatibility between this mod and epic fight mod"?

**Comment by megagnarxd** [comment]
I've never coded so bear with me. I will try to explain it to the best of my abilities.

Yes, the original reskillable used experience points instead of levels. So when you are level 30 and leveling up a level 3 skill, you will not go from 30-3=27.

Epic fight mod uses a different attacking type. I have no idea how it works but I'm assuming reskillable register a hit as left clicking. Projectmmo, however, is able to stop all of epic fight's attacks. You may want to look into projectmmo's code and see how they do it. Thank you

**Comment by Floris0106** [comment]
I don't know whether I want the mod to work like that with xp, but I'll consider adding that as a config option. As for the mod compatibility, I don't know whether I want to work on these kinds of features.

**Comment by Floris0106** [comment]
I've decided to start implementing the XP usage change, but I won't be implementing this mod compat. Anyone willing to put in the work can submit a pull request however.

**Comment by Floris0106** [comment]
I've implemented the option to change the usage of xp, the new version will be on curseforge soon!

**Comment by megagnarxd** [comment]
very nice. Thank you very much

**Comment by megagnarxd** [comment]
https://user-images.githubusercontent.com/92188308/144981066-4aa63d2b-e573-4541-9013-f07306812c50.mp4

Hey, it doesn't work if you enable exp

**Comment by megagnarxd** [comment]
Ah, it doesn't work if you use the command.

**Comment by megagnarxd** [comment]
Doesn't work if you do /experience add levels. But that's fine. Good work

**Comment by Floris0106** [comment]
What do you mean by that exactly?

**Comment by megagnarxd** [comment]
/experience add levels
the command, doesn't work. But otherwise, it's all good. Thank you for making this happen

**Comment by Floris0106** [comment]
I don't understand exactly what you mean when you say it doesn't work. Do you mean using the command doesn't give you levels?

**Comment by megagnarxd** [comment]
No, /experience command does give level. However, you can't use said experience to level up your reskillable skills

**Comment by Floris0106** [comment]
Hmm, I'll look into that, not sure why that is.

**Comment by Jollyolsport** [comment]
Maybe instead of having all levels drop back to zero, only a couple levels will be reduced by a small random amount after every death? Much like the first reskillable?

**Comment by Floris0106** [comment]
I'll reopen your previous issue, this issue is just poorly named and not for all feature suggestions.
